### DISCUSSION TITLE
Frequency Capturing

### DISCUSSION THREAD

**Discussion by dousamark** [szimek/signature_pad#611] [Q&A]
Hello,

Drawing looks smooth at my monitor, but I would like to ask at what frequency do you capture points? More specifically how many points can you capture per minute when we assume signing on a new device, so that hardware wouldn't be a bottle neck.

Thanks in advance

**Comment by UziTech** [comment]
I believe this depends on the browser. We use `mousemove` and similar events to determine interaction. So however fast the browser sends those events.
